wiki/Help_Centre/en.md

```diff
@@ -157,41 +179,52 @@ If we were to allow everybody to make and play on multiple accounts, the ranked
 
 Other games encounter this issue a lot with what is commonly known as "smurfs" - accounts with low playtimes and totals that inflate the rankings and prevent other users from playing competitively.
 
-In osu!, we are very much against users having multiple accounts, and we enforce a strict rule of only one account per person, ever.
+In osu!, we are very much against users having multiple accounts, and we enforce a strict rule of only one account per person, ever. You get to play on the first account you ever registered with us, and that's it!
 
-You get to play on the first account you ever registered with us, and that's it!
+If you currently have more than one account, please contact [accounts@ppy.sh](mailto:accounts@ppy.sh) to get things sorted out.
 
-#### Is it possible to start over or have my account's statistics wiped?
+#### Is it possible to start over/have my account's statistics wiped/have my account deleted?
 
 **Not at the moment.**
 
 We are considering extra 'score reset' purchases from the osu!store for the truly hardcore players who wish nothing but perfection in their stats, but as it currently stands, we cannot wipe an account's statistics for you.
 
-This doesn't mean you can make a new account to do the same thing, either! Having multiple accounts is strongly frowned upon in our community and we will disable any such accounts that are brought to our attention.
+This doesn't mean you can make a new account to do the same thing, either! Having multiple accounts is against the rules, and we will disable any such accounts that are brought to our attention.
+
+If you are covered by the European Union, then you have the option to delete your account. You can read more about this on the [Privacy Page](https://osu.ppy.sh/legal/en/Privacy#your-rights-and-control). If you are not covered by the European Union, then we will not delete your account.
```

Comment:
this should be under a new header titled "Can I delete my account?"

all users can file a deletion request, even those not covered by the GDPR (this is a recent change and needs to be propagated throughout other articles as well)